src/messages/index.ts

```diff
@@ -21,4 +23,9 @@ export const messages = {
   enabled: 'enabled',
   disabled: 'disabled',
   restart: '\'Restart\' for changes to take effect.',
+  ng2Detected: 'VSCode-Icons has detected an Angular 2 project. ' +
```

Comment:
There are plenty of links in the internet about this matter. [This is is just an example](http://angularjs.blogspot.com.es/2016/12/ok-let-me-explain-its-going-to-be.html)